modules/quotas-creating-object-count-quotas.adoc

```diff
@@ -5,13 +5,10 @@
 [id="quota-creating-object-count-quotas_{context}"]
 = Creating object count quotas
 
-You can create an object count quota for all {product-title} standard namespaced
-resource types, such as `BuildConfig`, and `DeploymentConfig`. An object quota
+You can create an object count quota for all {product-title} standard namespaced resource types, such as `BuildConfig` and `DeploymentConfig`. An object quota
```

Comment:
For readability, how about saying "standard namespaced resource types on {product-title}, such as..."? It breaks up the adjectives a little bit. :)
Also, based on our [object refs guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-object-references), I would change:

s/such as `BuildConfig` and `DeploymentConfig`./such as `BuildConfig` and `DeploymentConfig` objects.